xml/System.Text.Encodings.Web/HtmlEncoder.xml

```diff
@@ -82,10 +82,16 @@
         </Parameter>
       </Parameters>
       <Docs>
-        <param name="allowedRanges">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="allowedRanges">Set of characters that the encoder is allowed to not encode.</param>
+        <summary>Creates a new instance of HtmlEncoder specifying characters to be encoded.</summary>
+        <returns>A new instance of the <see cref="HtmlEncoder"/></returns>
+        <remarks><format type="text/markdown"><![CDATA[ 
+
+## Remarks  
+ Some characters in <paramref name="allowedRanges"/> might still get encoded, i.e. this parameter is just telling the encoder what ranges it is allowed to not encode, not what characters it must not encode.  
```

Comment:
get encoded --> be encoded
is just telling...-> indicates what ranges the encoder is allowed to not encode, not what characters it must not encode.